### DISCUSSION TITLE
Crash after pausing

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#24455] [Q&A]
### Type

Crash to desktop

### Bug description

The crash occurred slightly after pausing (I think) on the beatmap [Yousei Teikoku - Kuusou Mesorogiwi \[Insane\]](https://lazer.ppy.sh/beatmapsets/39031#osu/124701). The game had been behaving normally up until this point.

### Version

2023.717.0-lazer

### Logs

[updater.log](https://github.com/ppy/osu/files/12245659/updater.log)
[database.log](https://github.com/ppy/osu/files/12245660/database.log)
[input.log](https://github.com/ppy/osu/files/12245661/input.log)
[network.log](https://github.com/ppy/osu/files/12245662/network.log)
[performance.log](https://github.com/ppy/osu/files/12245663/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12245664/runtime.log)

**Comment by peppy** [comment]
The logs show no crash. This usually means it's a GPU driver crash or otherwise.

**Reply by Non-Binary-Programmer** [reply]
is there anything I need to do with my drivers, then?
